ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html

```diff
@@ -54,6 +54,19 @@ <h3 class="cmp-accordion__header">
                 <span class="cmp-accordion__title">${accordion.exportedItems[item].label.value}</span>
                 <span class="cmp-accordion__icon"></span>
             </button>
+            <div 
+              data-sly-test="${accordion.exportedItems[item].repeatable}" 
+              class="cmp-accordion__repeatable-buttons"
+              data-cmp-hook-adaptiveFormAccordion="repeatableButton">
```

Comment:
Add test for these buttons:
simple case and for repeatable accordion inside repeatable accordion.